pkg/arvo/app/graph-store.hoon

```diff
@@ -5,9 +5,9 @@
 |%
 +$  card  card:agent:gall
 +$  versioned-state
-  $%  [%0 network:zero:store]
-      [%1 network:zero:store]
-      [%2 network:zero:store]
+  $%  [%0 *]
+      [%1 *]
+      [%2 *]
```

Comment:
These are all pre breach, yeah?

Reply:
%2 is not, but %0 and %1 are. Would you like me to add back %2?